courses/machine_learning/deepdive2/structured/solutions/3a_bqml_baseline_babyweight.ipynb

```diff
@@ -0,0 +1,616 @@
+{
```

Comment:
<ul><li>I am not understanding what the following sentence is trying to convey. I suggest we remove it: "Recall that a model in BigQuery ML represents what an ML system has learned from the training data."</li><li>Since a baseline can also be implemented by a ml model (as is done below with the linreg model), I suggest we also remove: "A baseline model is a solution to a problem without applying any machine learning techniques."</li><li>Baseline Model: Create the baseline model -&gt; "Create the baseline model"</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/668/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Done</p>